controllers/pod.go

```diff
@@ -399,6 +405,12 @@ func (r *SingleClusterReconciler) deletePodAndEnsureImageUpdated(
 
 	// Delete pods
 	for _, p := range podsToUpdate {
+		if r.aeroCluster.Spec.CleanLocalPVC {
+			if err := r.deleteLocalPVCs(p); err != nil {
+				return reconcileError(err)
+			}
+		}
+
```

Comment:
What's the use-case of adding this deletePVC change in upgrade flow?

Reply:
yeah we can remove `deletePVC` from upgrade flow.